pkg/sidecar/config/config.go

```diff
@@ -34,6 +30,7 @@ const (
 	configDirScyllaD                    = "/etc/scylla.d"
 	scyllaYAMLPath                      = configDirScylla + "/" + naming.ScyllaConfigName
 	scyllaYAMLConfigMapPath             = naming.ScyllaConfigDirName + "/" + naming.ScyllaConfigName
+	scyllaYAMLManagedConfigMapPath      = naming.ScyllaDBManagedConfigDir + "/scylladb-managed-config.yaml"
```

Comment:
`scylladb-managed-config.yaml` could be const, and then synchronized between this variable and actual ConfigMap content.

Reply:
done